nss-day-cohort-22/Class-Website
# Hey Guys

## Description of Proposed Changes
Adding my info and Added some more tech images and descriptions for some of the backend stuff
- 

## Steps to Test

First ask yourself, Can you handle this pull request? :shipit: 


```sh
git fetch --all
git checkout GW_Stuff

```
THERE SHOULD BE LOGOS AND TEXT FOR MVC, CORE 2.0, VISUAL STUDIO, UGLIFY, AND ESLINT

fuck it we used it right? makes us look fancy. :pizza: :tada:  

## Impacted Areas in Application

List general components of the application that this PR will affect:
muh bio

tech imagez
* 

## Mentions @username
@krnorris65 
Tag users that need to review this code
